templates/repo/view_list.tmpl

```diff
@@ -83,7 +83,7 @@
 				{{end}}
 				<td class="message">
 					<span class="truncate has-emoji">
-						<a href="{{$.RepoLink}}/commit/{{$commit.ID}}" title="{{$commit.Summary}}">{{$commit.Summary}}</a>
+						{{RenderCommitMessageLink $commit.Message $.RepoLink (Printf "%s/commit/%s" $.RepoLink $commit.ID)  $.Repository.ComposeMetas}}
```

Comment:
I don't think this is good idea to format this as there is not much point in this as links with in links will add more confusion

Reply:
Maybe what we need is different styling on the links inside links to make it more clear to users.

Reply:
If I remember correctly there was problems with links within links already in previous versions

Reply:
imho this line change should be currently reverted until we come up with good solution for this

Reply:
What were the issues? I'd like to take a look as to whether they can be addressed.

Reply:
First of all you can really tell apart where inner links begin and outer ends.
Need to check also if overflow hiding and elipsis adding does not breaks everyting if inner link happens to be at the end.

Yes, please open PR

Reply:
@lafriks #7923 

I think we can also use the PR to get some info about what the issues with the feature were.